### PR TITLE
RFE-157 - Add Edit profile page layout

### PR DESCRIPTION
**Pull request by adam7918**
**Summary**

- Created the editprofile.js page inside of the /page directory

- Added structure for profile editing form.

- Added redirection for /user and /editprofile if the user is unauthorized.

### CODE REVIEW

File: app/server.js [hunk 1]
```diff
@@ -23,6 +23,25 @@ app
       }
     });
 
+    server.get("/editprofile", (req, res) => {
```
Comment (Kingdutch) [inline app/server.js]: We use the following URL for this:

```suggestion
    server.get("/user/<userid>/profile", (req, res) => {
```

When building this, think about what happens when a user edits a profile of another user.

File: app/pages/editprofile.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+  };
+
+  return (
+    <Layout title="Edit profile | Open Social">
+      <StyledTitle>Edit Profile</StyledTitle>
+      <Form onSubmit={handleSubmit}>
+        {/* Names and profile image card */}
+        <StyledCard>
+          <CardHeader>
+            <div>Names and profile image</div>
```
Comment (Kingdutch) [inline app/pages/editprofile.js]: This div isn't needed here I think ? :)

File: app/pages/editprofile.js [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+          </CardHeader>
+          <CardBody>
+            <BlockFormField>
+              <Label>First name</Label>
+              <Input type="text" name="firstname"></Input>
+            </BlockFormField>
```
Comment (Kingdutch) [inline app/pages/editprofile.js]: Cool :)

File: app/pages/editprofile.js [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+              <InputDescription>
+                Separate multiple values by a comma.
+              </InputDescription>
+            </BlockFormField>
+          </CardBody>
+        </StyledCard>
+        <FormButtons>
+          <Link href="/user">
+            <TextButton>Cancel</TextButton>
+          </Link>
```
Comment (Kingdutch) [inline app/pages/editprofile.js]: A link within a button doesn't see right (what happens, is the button clicked or the link pressed?).

Perhaps you want `<TextButton as={Link}>` instead?

File: app/server.js (hunk 1 above)
Comment (adam7918) [reply app/server.js]: I will fix this on a different branch as it is part of [RFE-166](https://getopensocial.atlassian.net/jira/software/projects/RFE/boards/109?selectedIssue=RFE-166).

**Pull request merged**